packages/date-picker/src/CalendarNav.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library"
+import arrowRight from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import arrowLeft from "@kaizen/component-library/icons/arrow-left.icon.svg"
+import styles from "./DatePicker.scss"
+
+export type CalendarNavProps = {
+  onPreviousClick: () => void
```

Comment:
/nit does `onPreviousClick` and `onNextClick` need to be documented in storybook? I can't see them in the props table.

Reply:
Or are these not available to the consumer? I think there's a lot of value in exposing them.

Reply:
I know naming is hard and contentious ;) but imho `onPreviousClick/onNextClick` lack some context, maybe consider something like `onPreviousMonthClick` which is bit wordier but also clearer.

Reply:
Was leaving this kind of nitty gritty for the post-styling PR but happy to add these in as props with this PR.

Reply:
I'm going to leave this for the next PR where we expose more of the API for consumers. This is mostly the styling/basic API to get a base in to work from.